public/index.html

```diff
@@ -7,6 +7,13 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+
```

Comment:
@pankajsahu221 Why are you adding bootstrap 5 css link here? If you want to use bootstrap 5 then install it using npm and import it in the component.
Also see my below comment related to semantic-ui, I think we don't need to use react-bootstrap here.

Reply:
Okay, I'll fix that.